mappings/net/minecraft/world/biome/layer/BiomeLayerMapToValidBiomes.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS none/azc net/minecraft/world/biome/layer/BiomeLayerMapToValidBiomes
+	FIELD c biomesHot [Lnone/ajj;
+	FIELD d biomesTrees [Lnone/ajj;
+	FIELD e biomesCold [Lnone/ajj;
+	FIELD f biomesFrozen [Lnone/ajj;
+	FIELD g biomeGeneratorSettings Lnone/auq;
+	METHOD <init> (JLnone/azi;Lnone/ajb;Ljava/lang/String;)V
+		ARG 0 seed
+		ARG 1 previous
+		ARG 2 levelGeneratorType
+		ARG 3 generatorOptions
```

Comment:
Is this assigned to `biomeGeneratorSettings` ? Should probably either be `level` or `biomeGeneratorSettings`. Let's not call them `options` ..?